src/scss/pages/_repo_detail.scss

```diff
@@ -0,0 +1,22 @@
+.detail-container {
```

Comment:
@craigmj Yes please this is still WIP hence the tasks.  Will move the card on the board when ready for review